docs/src/process/testing/test-heuristics.md

```diff
@@ -0,0 +1,73 @@
+# Test Heuristics / QA Checklist
+
+The purpose of this document is to identify common patterns and "implicit requirements" associated with Open MCT domain objects to prevent bugs as early as possible in the SDLC: the design and implementation phases. 
+
+These checklist items are not strict rules to follow, but just a list of "implicit requirements" that come when adding new domain objects to Open MCT given how it is leveraged by many downstream projects. When a checklist item in the form of a question, they should be viewed as design decision as well as a requirement.
+
+This technique is well established and is found in https://pragprog.com/titles/ehxta/explore-it/
+## Net New Domain Object
+The following checklist items should be considered when adding a net new domain object.
+
+### CRUD
+Checklist items specific to creating, reading, updating, or deleting interactions.
+
+#### Create
```

Comment:
I think we may want a note similar to, "For domain objects that have the `creatable` property set to `true`"